app/utilities.py

```diff
@@ -0,0 +1,12 @@
+def validate_request_body(request_body, list_of_attributes):
```

Comment:
I left a comment above about creating a function just like this!! Think about all of the other places you could've used this helper function 👀